docs: grammatical improvements (#1738)
nuxt/nuxtjs.org: content/en/index.md

::section-content-item
---
title: SEO Friendly

> description: 'Meta Tags management and faster time-to-content for great indexing.'

Reviewer: ```suggestion
description: 'Meta tag management and faster time-to-content for great indexing.'
```

Sustainable [_Development_]{.text-primary}

#description

> Nuxt development is made from passionate developers, however, the amount of effort needed to maintain and develop new features is not sustainable without proper financial backing. We are thankful for our sponsors and partners that help make Nuxt possible.<br>

Reviewer: ```suggestion
Nuxt development is carried out by passionate developers, but the amount of effort needed to maintain and develop new features is not sustainable without proper financial backing. We are thankful for our sponsors and partners, who help make Nuxt possible.<br>
```

> Nuxt development is made from passionate developers, however, the amount of effort needed to maintain and develop new features is not sustainable without proper financial backing. We are thankful for our sponsors and partners that help make Nuxt possible.<br>

Reviewer: ```suggestion
Nuxt development is carried out by passionate developers, but the amount of effort needed to maintain and develop new features is not sustainable without proper financial backing. We are thankful for our sponsors and partners, who help make Nuxt possible.<br>
```

Sharing is [Caring]{.text-sky-lighter}

#description

> Discover articles from the framework Team and Community about Nuxt. Tips and tricks included!

Reviewer: ```suggestion
Discover articles from the framework team and community about Nuxt. Tips and tricks included!
```
